template-parts/content-blocks/block-carousel.php

```diff
@@ -0,0 +1,63 @@
+<?php
+/**
+ * The template used for displaying a caroursel.
+ *
+ * @package _s
+ */
+
+// Start repeater markup...
+if ( have_rows( 'carousel_slides' ) ) :
+	echo '<section class="content-block container carousel">';
+
+	// Enqueue Slick carousel.
+	wp_enqueue_style( 'slick-carousel' );
+	wp_enqueue_script( 'slick-carousel' );
+
+	// Loop through slide slides.
+	while ( have_rows( 'carousel_slides' ) ) :
+		the_row();
+
+		// Set up fields.
+		$title           = get_sub_field( 'title' );
+		$text            = get_sub_field( 'text' );
+		$button_text     = get_sub_field( 'button_text' );
+		$button_url      = get_sub_field( 'button_url' );
+		$animation_class = _s_get_animation_class();
+		$other_options   = get_sub_field( 'other_options' ) ? get_sub_field( 'other_options' ) : get_field( 'other_options' )['other_options'];
+
+		// If the block has expired, then bail!
+		if ( _s_has_block_expired( array(
+			'start_date' => $other_options['start_date'],
+			'end_date'   => $other_options['end_date'],
+		) ) ) {
+			return false;
```

Comment:
Because we're returning here, if I have an expired slide in the middle of a group of slides I don't get anything after the expired slide.

Example:
I have 3 slides in a carousel. I set the 2nd slide to be expired. Now my carousel only displays the first slide.

Reply:
Fixed.